DMTN-165.tex

```diff
@@ -132,16 +132,24 @@ \section{The Hybrid Alert Packet Concept}
 They are not meant to completely characterize any event, but merely to provide maximum scientific distinguishing power, to allow users to determine if this alert is likely a transient, variable, or moving object.
 It is expected that users will still need to retrieve the full alert packet to be able to achieve their science goals.
 
-The size of the lightweight alert is $\sim$200 bytes/packet. 
+The size of the lightweight alert is $\sim$200 bytes/packet.
 Averaged over 39 seconds per visit this implies an average bandwidth required of 400 kbps; to transmit all of the lightweight alerts in a visit within 5 seconds requires 3.2 Mbps.
 This implies that we can easily serve thousands of users the lightweight alert stream using a 10 Gbps network interface from the Data Facility.
 A night of observing would produce a total volume of lightweight alerts of 2\,GB.
 In contrast to the full alert stream, which is $\sim$ 820\,GB per night, it is easy to imagine a user accessing the full alert stream from their laptop, or perhaps connecting a small or ad-hoc automated telescope to it.
 We believe that democratizing access to the alert stream will enable some of the most creative uses of the stream and ensure innovative use by scientists.
 
+Users can fetch the full alert packet, but these requests would be rate limited to keep them from unfairly consuming all available bandwidth.
+Choosing an appropriate value for the rate limit is a policy decision which must balance the science benefits of letting users access full packets against providing access to more users.
+
+For example, the rate limit could be set to permit users to get all data as long as they spread their requests out.
+To keep up with an event generation rate of 10,000 events per 39 second exposure, users would need to retrieve 270 events per second; at 80\,KB per event, this would require 164 Mbps of bandwidth per user.
+At this volume, we could support 60 users within the 10 Gbps bandwidth budget.
+Tighter rate limits would permit more users, but those users would have to choose to receive only a subset of the full alert payloads.
```

Comment:
```suggestion
Tighter rate limits would permit more users, but those users would have to choose to receive only a subset of the available full alerts.
```

Reply:
I suggest a slight change here--otherwise it could be interpreted as allowing users to grab portions of all alerts, which I don't think we're planning.

Reply:
Very good point.